nsxt/policy_utils.go

```diff
@@ -382,6 +382,20 @@ func nsxtPolicyPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*s
 	return rd, nil
 }
 
+func nsxtVPCPathResourceImporter(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	rd, err := nsxtPolicyPathResourceImporterHelper(d, m)
+	if errors.Is(err, ErrNotAPolicyPath) {
+		return rd, nil
```

Comment:
I see this behavior in other importer as well - could you please remind me why we swallow `ErrNotAPolicyPath` error?

Reply:
In importers which support importing via both `path` and `id`, it's used to handle the `id` case. In this case it's a bug, I'll fix this.